test/dashjs.test.js

```diff
@@ -92,12 +92,16 @@
               setLimitBitrateByPortal: function(value) {
                 setLimitBitrateByPortalCalled = true;
                 setLimitBitrateByPortalValue = value;
-              }
+              },
+
+              on: function() {}
             };
           }
         };
       };
 
+      dashjs.MediaPlayer.events = { ERROR: '' };
```

Comment:
`ERROR` itself doesn't need to exist for the tests but `dashjs.MediaPlayer.events` does